### PR TITLE
Add GreatEyes detector

### PR DESCRIPTION
**Pull request by EliotGann**
Implemented with @mrakitin at NSLS-II RSOXS beamline.

### CODE REVIEW

File: ophyd/areadetector/detectors.py [hunk 1]
```diff
@@ -125,6 +126,11 @@ class FirewireWinDetector(DetectorBase):
     cam = C(cam.FirewireWinDetectorCam, 'cam1:')
 
 
+class GreatEyesDetector(DetectorBase):
+    _html_docs = ['greateyesDoc.html']  # the documentation is not public
```
Comment (klauer) [inline ophyd/areadetector/detectors.py]:
```suggestion
    _html_docs = []  # the documentation is not public
```

The documentation is used internally to get docstrings for AreaDetector components, so clearing this list would be appropriate.

Everything else looks fine. Thanks!

Comment (mrakitin) [reply ophyd/areadetector/detectors.py]: Thanks @klauer! I tried to apply the suggestion, but don't seem to have permissions to push to @EliotGann's PR branch.

@EliotGann, can you please accept Ken's suggestions on this page? That will go as a separate commit into your branch.

**Comment by mrakitin** [comment]
For those who are interested in the implementation at the beamline, here is the link: https://github.com/NSLS-II-SST/profile_collection/commit/1883eb5496662adf65fa5373cc21af6edfd2cc09.

File: ophyd/areadetector/detectors.py (hunk 1 above)
Comment (EliotGann) [reply ophyd/areadetector/detectors.py]: I tried to accept the suggestion ... not sure if I did it correctly

Comment (mrakitin) [reply ophyd/areadetector/detectors.py]: Yeah, it was correctly done, we can see 583bb9b1181905fe04c088d8db37272670fb402d commit now. Thanks!

**Comment by mrakitin** [comment]
I assume it's good to go given that we have 2 approvals.

**Pull request merged**